<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerGP</class>
 <widget class="QDialog" name="CustomerGP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="desc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Coupon Desc</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CMB_CouponStatus">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>420</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_createCoupon">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>450</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
                                        background-color: white;
                                        border-style: outset;
                                        border-width: 2px;
                                        border-radius: 10px;
                                        border-color: blue;
                                        min-width: 6em;
                                        }
                                        QPushButton:hover {
                                        background: qradialgradient(
                                        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
                                        );
                                        }
                                        QPushButton:pressed {
                                        border-style: inset;
                                        background: qradialgradient(
                                        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
                                        );
                                        }
                                    </string>
   </property>
   <property name="text">
    <string>Create Coupon</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_desc">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>271</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
                                        background-color: white;
                                        border-style: outset;
                                        border-width: 2px;
                                        border-radius: 10px;
                                        border-color: blue;
                                        min-width: 6em;
                                        }
                                        QLineEdit:hover {
                                        background: qradialgradient(
                                        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
                                        );
                                        }
                                       QLineEdit:pressed {
                                        border-style: inset;
                                        background: qradialgradient(
                                        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
                                        );
                                        }
                                    </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_desc_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>271</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
                                        background-color: white;
                                        border-style: outset;
                                        border-width: 2px;
                                        border-radius: 10px;
                                        border-color: blue;
                                        min-width: 6em;
                                        }
                                        QLineEdit:hover {
                                        background: qradialgradient(
                                        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
                                        );
                                        }
                                       QLineEdit:pressed {
                                        border-style: inset;
                                        background: qradialgradient(
                                        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
                                        );
                                        }
                                    </string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Value">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Discount Value</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_desc_3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>271</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
                                        background-color: white;
                                        border-style: outset;
                                        border-width: 2px;
                                        border-radius: 10px;
                                        border-color: blue;
                                        min-width: 6em;
                                        }
                                        QLineEdit:hover {
                                        background: qradialgradient(
                                        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
                                        );
                                        }
                                       QLineEdit:pressed {
                                        border-style: inset;
                                        background: qradialgradient(
                                        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
                                        );
                                        }
                                    </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Percentage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Discount Percentage</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Multi">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Multi Use Count</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_desc_4">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>260</y>
     <width>271</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
                                        background-color: white;
                                        border-style: outset;
                                        border-width: 2px;
                                        border-radius: 10px;
                                        border-color: blue;
                                        min-width: 6em;
                                        }
                                        QLineEdit:hover {
                                        background: qradialgradient(
                                        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
                                        );
                                        }
                                       QLineEdit:pressed {
                                        border-style: inset;
                                        background: qradialgradient(
                                        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
                                        );
                                        }
                                    </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_desc_5">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>300</y>
     <width>271</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
                                        background-color: white;
                                        border-style: outset;
                                        border-width: 2px;
                                        border-radius: 10px;
                                        border-color: blue;
                                        min-width: 6em;
                                        }
                                        QLineEdit:hover {
                                        background: qradialgradient(
                                        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
                                        );
                                        }
                                       QLineEdit:pressed {
                                        border-style: inset;
                                        background: qradialgradient(
                                        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
                                        );
                                        }
                                    </string>
   </property>
  </widget>
  <widget class="QLabel" name="desc_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Serial Count</string>
   </property>
  </widget>
  <widget class="QLabel" name="desc_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Valid From</string>
   </property>
  </widget>
  <widget class="QLabel" name="desc_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>360</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Valid To</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_desc_6">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>271</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
                                        background-color: white;
                                        border-style: outset;
                                        border-width: 2px;
                                        border-radius: 10px;
                                        border-color: blue;
                                        min-width: 6em;
                                        }
                                        QLineEdit:hover {
                                        background: qradialgradient(
                                        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
                                        );
                                        }
                                       QLineEdit:pressed {
                                        border-style: inset;
                                        background: qradialgradient(
                                        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
                                        );
                                        }
                                    </string>
   </property>
  </widget>
  <widget class="QLabel" name="desc_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Coupon ID</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="Qdate_to">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>360</y>
     <width>79</width>
     <height>18</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="minimumDateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2020</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="displayFormat">
    <string>d/M/yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="Qdate_from">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>360</y>
     <width>79</width>
     <height>18</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="minimumDateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2020</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="displayFormat">
    <string>d/M/yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="desc_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Company</string>
   </property>
  </widget>
  <widget class="QLabel" name="desc_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Branch</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
